pkg/networkservice/common/heal/client.go

```diff
@@ -193,6 +203,8 @@ func (u *healClient) listenToConnectionChanges(
 			return
 		}
 
+		logger.Infof("got event: %v", event)
```

Comment:
This probably can be too verbose for `.Info`.

Reply:
Oops, accidentally missed it.
Fixed.